content/connectors/examples/github.md

```diff
@@ -0,0 +1,379 @@
+  "svn_url": "https://github.com/infinyon/fluvio",
+  "homepage": "https://www.fluvio.io/",
+  "size": 14667,
+  "stargazers_count": 790,
+  "watchers_count": 790,
+  "language": "Rust",
+  "has_issues": true,
+  "has_projects": true,
+  "has_downloads": true,
+  "has_wiki": true,
+  "has_pages": false,
+  "forks_count": 50,
+  "mirror_url": null,
+  "archived": false,
+  "disabled": false,
+  "open_issues_count": 103,
+  "license": {
+    "key": "apache-2.0",
+    "name": "Apache License 2.0",
+    "spdx_id": "Apache-2.0",
+    "url": "https://api.github.com/licenses/apache-2.0",
+    "node_id": "MDc6TGljZW5zZTI="
+  },
+  "allow_forking": true,
+  "is_template": false,
+  "topics": [
+    "cloud-native",
+    "rust",
+    "rust-lang",
+    "streaming"
+  ],
+  "visibility": "public",
+  "forks": 50,
+  "open_issues": 103,
+  "watchers": 790,
+  "default_branch": "master",
+  "temp_clone_token": null,
+  "organization": {
+    "login": "infinyon",
+    "id": 52172389,
+    "node_id": "MDEyOk9yZ2FuaXphdGlvbjUyMTcyMzg5",
+    "avatar_url": "https://avatars.githubusercontent.com/u/52172389?v=4",
+    "gravatar_id": "",
+    "url": "https://api.github.com/users/infinyon",
+    "html_url": "https://github.com/infinyon",
+    "followers_url": "https://api.github.com/users/infinyon/followers",
+    "following_url": "https://api.github.com/users/infinyon/following{/other_user}",
+    "gists_url": "https://api.github.com/users/infinyon/gists{/gist_id}",
+    "starred_url": "https://api.github.com/users/infinyon/starred{/owner}{/repo}",
+    "subscriptions_url": "https://api.github.com/users/infinyon/subscriptions",
+    "organizations_url": "https://api.github.com/users/infinyon/orgs",
+    "repos_url": "https://api.github.com/users/infinyon/repos",
+    "events_url": "https://api.github.com/users/infinyon/events{/privacy}",
+    "received_events_url": "https://api.github.com/users/infinyon/received_events",
+    "type": "Organization",
+    "site_admin": false
+  },
+  "network_count": 50,
+  "subscribers_count": 21
+}
+```
+
+As we can see, the data we want is in here (`stargazers_count` and `forks_count`),
+but it's buried under the slew of other fields that we don't need for our use-case. If we kept a
+long-lived stream of this data running, we'd be using hundreds of times the bandwidth necessary.
+
+Let's see how we can use a custom SmartModule to select just these three fields we're interested
+in to create a lean and lightweight stream that meets our needs.
+
+### Using a SmartModule to shape GitHub data
+
+Now we're going to take a look at how a SmartModule can help us slim down our HTTP responses.
+We'll be using [this example GitHub SmartModule][2], which we'll learn more about in this section.
+If you'd like to create your own SmartModule from scratch, see [the docs on SmartModule Maps][3].
+
+Since we'll be using a pre-written SmartModule, the first thing we need to do is clone the repository:
```

Comment:
Can we build it from scratch instead? We can use the github project as an alternative.
The only difference I see is the `Cargo` file.